repair/row_level.cc

```diff
@@ -1063,10 +1063,12 @@ class repair_meta {
     }
 
     future<> set_estimated_partitions(uint64_t estimated_partitions) {
-        return with_gate(_gate, [this, estimated_partitions] {
```

Comment:
It looks like this method is returning future just to make a nice tail-call in `repair_set_estimated_partitions()` for local myip in there